### ISSUE TITLE
Standard model evaluation

### ISSUE THREAD

**Issue by SamVanhoutte** [arcus-azure/arcus.ml#14]
Pass a model with a test set (X + y) to the function and have this function output the evaluation metrics.

Output standard metrics
- All: accuracy
- Classifiers: ROC curve, Confusion matrix

__To be defined__:
- Should we have different functions for regression, compared to classification (binary / multiclass) ?

**Comment by SamVanhoutte** [comment]
Existing samples from notebooks I've built:

```python
def EvaluateModel(model, X_test_input, y_expected_output):
    # Evaluatie van de classifier
    y_predicted = model.predict(X_test_input)

    # Confusion matrix
    print(confusion_matrix(y_expected_output, y_predicted))

    # Classification report
    print(classification_report(y_expected_output, y_predicted))
    print(accuracy_score(y_expected_output, y_predicted))
```

```python

def EvaluateModel(selected_model, X_test_features, y_test_output, show_roc = False):
    y_pred = selected_model.predict(X_test_features)
    print(classification_report(y_test_output, y_pred))

    cf = confusion_matrix(y_test_output, y_pred)
    print(cf)
    accuracy = accuracy_score(y_test_output, y_pred) * 100
    print('Accuracy score:', accuracy) 
    if(show_roc == True):
        ShowRoc(y_test_output, y_pred)        

def ShowRoc(y_test_output, y_pred):
    import sklearn.metrics as metrics
    # calculate the fpr and tpr for all thresholds of the classification
    fpr, tpr, threshold = metrics.roc_curve(y_test_output, y_pred)
    roc_auc = metrics.auc(fpr, tpr)

    # method I: plt
    import matplotlib.pyplot as plt
    plt.title('Receiver Operating Characteristic')
    plt.plot(fpr, tpr, 'b', label = 'AUC = %0.2f' % roc_auc)
    plt.legend(loc = 'lower right')
    plt.plot([0, 1], [0, 1],'r--')
    plt.xlim([0, 1])
    plt.ylim([0, 1])
    plt.ylabel('True Positive Rate')
    plt.xlabel('False Positive Rate')
    plt.show()
```
